Tests/aweXpect.Reflection.Tests/ThatConstructor.IsStatic.Tests.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System.Linq;
+using System.Reflection;
+using aweXpect.Reflection.Tests.TestHelpers.Types;
+
+namespace aweXpect.Reflection.Tests;
+
+public sealed partial class ThatConstructor
+{
+	public sealed class IsStatic
+	{
+		public sealed class Tests
```

Comment:
Add negated test cases for all methods, so e.g. for
- `IsStatic.NegatedTests`
- `IsNotStatic.NegatedTests`
- `AreStatic.NegatedTests`
- `AreNotStatic.NegatedTests`

Please also add the negated tests for the static tests on type!